template/src/components/DebugMenu.js

```diff
@@ -0,0 +1,259 @@
+      <View style={styles.content}>
+        {infos.map(({title, description}) => (
+          <InfoMenuRow style={styles.infoMenu} title={title} description={description} />
+        ))}
+      </View>
+    </View>
+  )
+}
+
+const EnvironmentSection = ({title, children}) => (
+  <View style={styles.section}>
+    <Text style={styles.h3}>{title}</Text>
+    <View style={styles.content}>{children}</View>
+  </View>
+)
+
+export const DebugMenu = () => {
+  const [modalVisible, setModalVisible] = useState(false)
+  const openModal = useCallback(() => setModalVisible(true), [])
+  const closeModal = useCallback(() => setModalVisible(false), [])
+
+  const dispatch = useDispatch()
+
+  const codePushKey = useSelector(getCodePushKey)
+
+  const currentApiUrl = useSelector(getApiUrl)
+
+  const [bottomSheetType, setBottomSheetType] = useState()
+
+  const dimensions = useWindowDimensions()
+
+  const bottomSheetRef = useRef(null)
+
+  const snapPoints = useMemo(() => ['95%'], [])
+
+  const handleSnapPress = useCallback(index => {
+    bottomSheetRef.current?.snapToIndex(index)
+  }, [])
+  const handleClosePress = useCallback(() => {
+    bottomSheetRef.current?.close()
+  }, [])
+
+  const openEnvironmentBottomSheet = useCallback(() => {
+    setBottomSheetType(BOTTOM_SHEET_TYPE.env)
+    closeModal()
+    handleSnapPress(0)
+  }, [closeModal, handleSnapPress])
+
+  const openCodePushBottomSheet = useCallback(() => {
+    setBottomSheetType(BOTTOM_SHEET_TYPE.codePush)
+    closeModal()
+    handleSnapPress(0)
+  }, [closeModal, handleSnapPress])
+
+  const renderEnvironmentItem = useCallback(
+    ({item}) => {
+      const isActive =
+        bottomSheetType === BOTTOM_SHEET_TYPE.env ? item === currentApiUrl : item.dev === codePushKey
+
+      const onPress = () => {
+        if (bottomSheetType === BOTTOM_SHEET_TYPE.codePush) {
+          dispatch(appActions.setCodePushKey(item.dev))
+          //CodePush.restartApp(false)
+        } else {
+          dispatch(appActions.setApiUrl(item))
+        }
+        handleClosePress()
+      }
+      return (
+        <Pressable disabled={isActive} style={styles.flatListItem} onPress={onPress}>
+          <Text
+            color={isActive ? colors.primary : colors.black}
+            fontWeight={isActive ? 'bold' : 'normal'}
+            style={styles.flatListItemTitle}>
+            {bottomSheetType === BOTTOM_SHEET_TYPE.env ? item : item.dev}
+          </Text>
+          {isActive ? <View style={styles.flatListItemIcon} /> : null}
+        </Pressable>
+      )
+    },
+    [bottomSheetType, codePushKey, currentApiUrl, dispatch, handleClosePress],
+  )
+
+  const codePush = useMemo(() => CODEPUSH_KEYS.find(item => item.dev === codePushKey), [codePushKey])
+
+  return (
+    <>
+      <Draggable
+        isCircle
+        renderColor={colors.primary}
+        renderSize={debugMenuSize}
+        animatedViewProps={{height: dimensions.height}}
```

Comment:
don't use style inline

Reply:
it should be flexible,
eslint isn't warning style inline